src/steps/devices/converter.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  createIntegrationEntity,
+  Entity,
+} from '@jupiterone/integration-sdk-core';
+import { HexnodeDevice } from '../../types';
+
+import { Entities } from '../constants';
+
+export function createDeviceEntity(device: HexnodeDevice): Entity {
+  return createIntegrationEntity({
+    entityData: {
+      source: device,
+      assign: {
+        _type: Entities.DEVICE._type,
+        _class: Entities.DEVICE._class,
+        _key: `${device.device_name}-${device.id.toString()}`,
```

Comment:
The key thing please :slightly_smiling_face: (e.g. `hexnode_device:${val}`) - again, maybe device.id would suffice :thinking: